tests/conftest.py

```diff
@@ -1250,6 +1251,29 @@ def translations_once() -> Generator[_patch]:
         patcher.stop()
 
 
+@pytest.fixture(autouse=True, scope="module")
+def evict_faked_translations(translations_once) -> Generator[_patch]:
```

Comment:
Do we need a test of this fixture?

Reply:
It would be nice, but I'm not sure how to write such a test?

Reply:
Maybe make two test modules with each a test that asserts that a domain is not in the translation cache at the start of the test and then sets up a mock integration of this domain. Both tests should use the same domain.

Reply:
Yes, but there's no guarantee those two modules will run in the same pytest process in CI.

Reply:
Ok. If we could force that somehow then.